### PR TITLE
Updating Faq document

### PR DESCRIPTION
**Pull request by divyatag**
### What does this PR do? What is the motivation?

Adding a solution section for updating service naming options. 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by urseberry — CHANGES_REQUESTED** [review]

File: content/en/tracing/troubleshooting/faq_apm.md [hunk 1]
```diff
@@ -66,7 +69,9 @@ For example, you might have a single service like `service:test` showing multipl
 - `service:test-mongodb`
 - `service:test-postgresdb`
 
-You can merge the service names using an environment variable like `DD_SERVICE_MAPPING` or `DD_TRACE_SERVICE_MAPPING`, depending on the language. 
+You can use [Inferred Service dependencies (beta)][18]. Inferred external APIs use the default naming scheme `net.peer.name`. For example: `api.stripe.com`, `api.twilio.com`, and `us6.api.mailchimp.com`. Inferred databases use the default naming `scheme db.instance`.
+
+Orm you can merge the service names using an environment variable like `DD_SERVICE_MAPPING` or `DD_TRACE_SERVICE_MAPPING`, depending on the language.
```
Comment (urseberry) [inline content/en/tracing/troubleshooting/faq_apm.md:74]:
```suggestion
Or, you can merge the service names using an environment variable like `DD_SERVICE_MAPPING` or `DD_TRACE_SERVICE_MAPPING`, depending on the language. 
```
Haha, sorry, I made a typo. "Orm" is not a word - I meant to add a comma.

**Review by urseberry — COMMENTED** [review]
The link target for [18] is still missing. At the bottom of the file, add a line like:
[18]: /relative/link/target

Targets within the public docs use relative links. Otherwise, use an absolute link.

File: content/en/tracing/troubleshooting/faq_apm.md (hunk 1 above)
Comment (iadjivon) [reply content/en/tracing/troubleshooting/faq_apm.md:74]: Hi Ursula, Thanks for your help on this! Do you need us to add the link on our end?

**Comment by iadjivon** [comment]
Changes made to include `[Inferred Service dependencies (beta)][10]` instead as the link for [10] is https://docs.datadoghq.com/tracing/guide/inferred-service-opt-in/?tab=java and should link as expected. 

Happy to answer any questions / make modification! :)

**Review by brett0000FF — APPROVED** [review]

**Pull request merged**